docusaurus/src/components/HeaderDecoration.jsx

```diff
@@ -93,33 +153,33 @@ const ConnectorMetadataCallout = ({
           <a href={github_url} target="_blank">
             {dockerImageTag}
           </a>
-          {lastUpdated && <span className={styles.deemphasizeText}>{`(Last updated ${dayjs(lastUpdated).fromNow()})`}</span>}
+          {lastUpdated && (
+            <span
+              className={styles.deemphasizeText}
+            >{`(Last updated ${dayjs(lastUpdated).fromNow()})`}</span>
+          )}
+        </MetadataStat>
+      )}
+      {cdkVersion && (
+        <MetadataStat label="CDK Version">
+          <a target="_blank" href={cdkVersionUrl}>
+            {cdkVersion}
+          </a>
+          {isLatestCDK && (
+            <span className={styles.deemphasizeText}>{"(Latest)"}</span>
+          )}
+        </MetadataStat>
+      )}
+      {syncSuccessRate && (
+        <MetadataStat label="Sync Success Rate">
+          <MetricIcon iconComponent={SUCCESS_ICON} level={syncSuccessRate} />
+        </MetadataStat>
+      )}
+      {usageRate && (
+        <MetadataStat label="Usage Rate">
+          <MetricIcon iconComponent={USAGE_ICON} level={usageRate} />
```

Comment:
non-blocking nit that can be addressed later: we should probably have a tooltip or make these icons a hyperlink to some page that explains what the symbols mean, since a newcomer to the docs won't really understand the different icons' meanings